editor/blocks/list/index.js

```diff
@@ -53,29 +57,57 @@ wp.blocks.registerBlock( 'core/list', {
 			onClick( attributes, setAttributes ) {
 				setAttributes( { align: 'justify' } );
 			}
+		},
+		{
+			icon: 'editor-ul',
+			title: wp.i18n.__( 'Convert to unordered' ),
+			isActive: ( { } ) => false, // To be implemented
+			onClick( attributes, setAttributes ) {
+				setAttributes( { listType: 'ul' } );
+			}
+		},
+		{
+			icon: 'editor-ol',
+			title: wp.i18n.__( 'Convert to ordered' ),
+			isActive: ( { } ) => false, // To be implemented
+			onClick( attributes, setAttributes ) {
+				setAttributes( { listType: 'ol' } );
+			}
 		}
+
 	],
 
-	edit( { attributes } ) {
+	edit( { attributes, setAttributes } ) {
 		const { listType = 'ol', items = [], align } = attributes;
 		const content = items.map( item => {
 			return `<li>${ item.value }</li>`;
 		} ).join( '' );
-
 		return (
 			<Editable
 				tagName={ listType }
 				style={ align ? { textAlign: align } : null }
+				onChange={ ( v, node ) => {
+					// The node is getting passed through from Tiny.
+					// Is there a way to parse this with hpq that doesn't
+					// involve a string?
+					const attrs = parse( '<' + node.nodeName + '>' + v + '</' + node.nodeName + '>', parseAttrs );
```

Comment:
We might not need to try so hard to pass this through `parseAttrs` again, if there's a more direct approach. Is there an easy way to turn `v` or `node` into an array of list items? We probably don't need `listType` parsed at this point at all. Could either be a `querySelector` on the node or a separate and more concise `hpq` parse:

```
setAttributes( parse( node, {
	items: query( 'li', html() )
} ) );
```

>Is there a way to parse this with hpq that doesn't involve a string?

In fact, hpq prefers a DOMElement and [under the hood converts a string](https://github.com/aduth/hpq/blob/c2ceef55b36f1bc7039d99c49e666d414cf07cf2/src/index.js#L19-L24).